Makefile.toml

```
@@ -0,0 +1,154 @@
+
+######################
+#
+# public tasks:
+# * stable-checks
+# * run-tests
+#
+######################
+
+[config]
+min_version = "0.32.2"
+default_to_workspace = false
+
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
+CARGO_MAKE_CLIPPY_ARGS = "-- --deny=warnings"
+CARGO_MAKE_CARGO_BUILD_TEST_FLAGS = ""
+
+[config.modify_core_tasks]
+private = true
+namespace = "core"
+
+[tasks.stable-checks]
+description = "Runs all validations for both workspaces on rust stable only."
+condition = { channels = [ "stable" ] }
+run_task = { name = [ "stable-checks-flow", "stable-checks-stdweb" ], fork = true }
+
+[tasks.run-tests]
+dependencies = ["tests-setup"]
+env = { CARGO_MAKE_WORKSPACE_SKIP_MEMBERS = ["examples/*", "yewtil/examples/*"] }
+run_task = { name = [ "run-tests-flow", "run-tests-stdweb", "tests-cleanup" ], fork = true }
+
+[tasks.stable-checks-stdweb]
+private = true
+cwd = "yew-stdweb"
+command = "cargo"
+args = [
+    "make",
+    "--loglevel",
+    "${CARGO_MAKE_LOG_LEVEL}",
+    "--profile",
+    "${CARGO_MAKE_PROFILE}",
+    "stable-checks-flow"
+]
+
+[tasks.stable-checks-flow]
+private = true
+workspace = true
+dependencies = [
+    "core::check-format-flow",
+    "core::check-flow",
+    "core::clippy-flow",
+    "post-stable-checks"
+]
+
+[tasks.post-stable-checks]
+
+[tasks.web-check]
+command = "cargo"
+args = ["web", "check", "--target", "wasm32-unknown-unknown"]
+
+[tasks.tests-setup]
+script = [
+'''
+#!@duckscript
+test_flags= array --headless --firefox
+yew_test_features= set wasm_test
+
+fn set_httpbin_test_features
+    HTTPBIN_URL = get_env HTTPBIN_URL
+    echo INFO: using ${HTTPBIN_URL} for fetch service tests
+    yew_test_features = set "${yew_test_features},httpbin_test"
+end
+
+if is_defined HTTPBIN_URL
+    set_httpbin_test_features
+else
+    start_docker = set true
+    args_provided = not is_empty ${CARGO_MAKE_TASK_ARGS}
+    if ${args_provided}
+        args = split ${CARGO_MAKE_TASK_ARGS} ;
+        for arg in ${args}
+            if eq ${arg} --skip-httpbin
+                start_docker = set false
+                release ${args}
+            end
+        end
+    end
+
+    if ${start_docker}
+        echo Starting httpbin docker image...
+        mkdir ./target
+        cidfile = set ./target/httpbin_container.cid
+
+        # if container already running, stop it
+        if is_file ${cidfile}
+            container_id = readfile ${cidfile}
+            rm ${cidfile}
+            set_env HTTPBIN_CONTAINER_ID ${container_id}
+            cm_run_task tests-cleanup
+        end
+
+        exec docker run -d --cidfile ${cidfile} -p 8080:8080 kennethreitz/httpbin
```

The httpbin server listens on port 80 (not 8080).

```suggestion
        exec docker run -d --cidfile ${cidfile} -p 8080:80 kennethreitz/httpbin
```

---

This can be resolved with the random port suggestion below.